src/preview/source.js

```diff
@@ -3,15 +3,91 @@ import reactElementToJSXString from 'react-element-to-jsx-string';
 import { PrismLight as SyntaxHighlighter } from 'react-syntax-highlighter';
 import { ghcolors } from 'react-syntax-highlighter/dist/esm/styles/prism';
 import jsx from 'react-syntax-highlighter/dist/esm/languages/prism/jsx';
+import styled from 'styled-components';
+import Box from '@sweatpants/box';
+import Button from '../ui/components/Button';
+import ScreenReaderOnly from '@sweatpants/screenreaderonly';
+import Orientation from '../ui/components/icons/Orientation';
+import IconWrapper from '../ui/components/IconWrapper';
+import Theme from '@sweatpants/theme';
+import { theme } from '../ui/theme';
+
+const StyledWrapper = styled(Box)`
+  user-select: ${(props) => (props.resizing ? 'none' : '')};
+  pointer-events: ${(props) => (props.resizing ? 'none' : '')};
+`;
+
+const StyledIcon = styled(Button)`
+  position: relative;
+  transform: ${(props) => (props.orientation === 'vertical' ? 'rotate(90deg)' : 'rotate(0deg)')};
+  transform-origin: 50% 50%;
+`;
+
+const StyledXResize = styled(Box)`
+  cursor: col-resize;
```

Comment:
What do you think about adding some visual hover state, or do you like the minimalism? Could maybe highlight the border, or show some sort of handle. Open to ideas!

Reply:
Yea hover state is a good idea, I'll add that in. I'll do something similar to what vscode does